docs/data/charts/bars/bars.md

```diff
@@ -28,9 +40,9 @@ For example `xAxis={[{ dataKey: 'x'}]}` or `series={[{ dataKey: 'y'}]}`.
 
 {{"demo": "BarsDataset.js"}}
 
-## Bar size
+### Bar size
 
-You can define bar dimensions with `categoryGapRatio` and `barGapRatio` properties.
+You can define bar dimensions with `categoryGapRatio` and `barGapRatio` props.
```

Comment:
Not really a "props" because it's properties of the axis values

```
<BarChart
  xAxis={[ { categoryGapRatio: 0.3, barGapRatio: 0.1 } ]}
/>
```

Reply:
Ah I see, thanks for clarifying! Props/properties can get pretty confusing in this context.

Reply:
WOuld you prefer we use "attribute" for this kind of stuff?